portfolio/src/main/webapp/script.js

```diff
@@ -143,10 +202,16 @@ function cleanData() {
 }
 
 /**
- * This function handles the visual refreshing of the quote list element 
+ * Handles the visual refreshing of the quote list element 
  * by setting its content to empty. 
  */
 function refreshQuotes() {
   const quoteListElement = document.getElementById('quoteWrapper');
   quoteListElement.innerHTML = '';
 }
+
+/** Creates a map and adds it to the page. */
+function createMap() {
+  const map = new google.maps.Map(document.getElementById('simpleMap'),
```

Comment:
I think it can be clearer to put the line break before document.

ex:

const map = new google.maps.Map(
  document...,
  {...});

Reply:
Somehow GitHub comment does not show the indentation correctly. Please indent according to the style guide.

Reply:
Done. In the commits that follow, I will have factored out all the JavaScript code for the map into a separate file, so the new commit might show that I don't have this function included anymore. I will make the change accordingly in the new JS file for the map, which will be pushed later with the map commits.